constructor/header.sh

```diff
@@ -450,7 +452,7 @@ for env_pkgs in "${PREFIX}"/pkgs/envs/*/; do
     if [ "$env_name" = "*" ]; then
         continue
     fi
-    printf "\nInstalling ${env_name} environment...\n\n"
+    printf "\nInstalling %s environment...\n\n" "${env_name}"
```

Comment:
I am curious why do we need `printf` instead of a simple `echo`? The `\n` newlines?

Reply:
There seems to be lots of reasons https://unix.stackexchange.com/questions/65803/why-is-printf-better-than-echo 

The reason this line was updated is due to https://www.shellcheck.net/wiki/SC2059

Reply:
Wow, what a rabbit hole. Thanks for the link. Yes, let's leave it like that 😂